src/components/FolderList.tsx

```diff
@@ -1,27 +1,33 @@
-import { useEffect, useState } from 'react';
+import { MouseEvent, useEffect, useState } from 'react';
 import { createPortal } from 'react-dom';
 import Modal from './modal/Modal';
 import changeNameImg from '../assets/folderChangeName.svg';
 import deleteImg from '../assets/folderDelete.svg';
 import sharedImg from '../assets/folderShared.svg';
 import { classNames, cond } from '../utils/style';
 import style from '../styles/FolderList.module.css';
+import { folderDataType } from '@/types/folderDataType.type';
+
+interface folderListProp {
```

Comment:
```suggestion
interface FolderListProp {
```
에러는 아니지만 타입 스크립트 컨벤션으로 타입은 파스칼 케이스를 사용할 것을 권장하고 있습니다.

[관련 문서](https://www.typescriptlang.org/docs/handbook/typescript-in-5-minutes.html) 공유드립니다.